src/odemis/acq/stream/_projection.py

```diff
@@ -1361,6 +1363,9 @@ def _on_selected_line(self, _):
     def _on_selected_time(self, _):
         self._shouldUpdateImage()
 
+    def _on_selected_angle(self, _):
+        self._shouldUpdateImage()
+
```

Comment:
This feels a bit like duplicated code since multiple methods just handle ignoring arguments. I understand it though.